chatrooms.lua

```diff
@@ -29,12 +35,22 @@ function OnStartup()
 				sDescription = "Chatroom for usage by VIPs.",
 				sEmail = "donot@mail.me"
 			},
-			sFileName = "VIPChat.txt"
-		}
+			sFileName = "VIPChat.txt",
+			tChatHistory = {},
+			tUsers = {},
+		},
 	}
-	for sIndex, sValue in pairs(tChatRooms) do
-		Core.RegBot( sIndex, sValue.BOT.sDescription, sValue.BOT.sEmail, true )
+	for sIndex, tValue in pairs( tChatRooms ) do
+		Core.RegBot( sIndex, tValue.BOT.sDescription, tValue.BOT.sEmail, true )
 	end
+	dofile( sPath..sExtPath.."offliner.lua" )
```

Comment:
You do not need to include `offliner.lua` file at all. You can see from line 19/23 (left/right in diff); that we are forwarding messages to everyone with profile no. `0` to `4`.